src/functions/util/addMessageFlags.js

```diff
@@ -0,0 +1,24 @@
+const { MessageFlags } = require("discord.js");
+
+module.exports = async (d) => {
+    const data = d.util.aoiFunc(d);
+    if (data.err) return d.error(data.err);
+
+    let [channelId, messageId, flags] = data.inside.splits;
+
+    flags = [flags];
```

Comment:
u can do ...flags and it will make it an array u know? this will also support multiple adds too